### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by tarasavage**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
...
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+
+  if (days >= 7) {
+    return days * dailyCost - 50;
+  }
+
+  if (days >= 3) {
+    return days * dailyCost - 20;
```
Comment (Viktor-Kost) [inline src/calculateRentalCost.js:38]: Try to follow DRY (Don't repeat yourself) principe. 
You have `days * dailyCost`. Just create a variable for it

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
GJ!
Just add a little improvement

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,18 @@
...
+  const dailyCost = 40;
+  const payment = days * dailyCost;
+
+  if (days >= 7) {
+    return payment - 50;
+  }
+
+  if (days >= 3) {
+    return payment - 20;
+  }
```
Comment (OlEzhA0) [inline src/calculateRentalCost.js:40]: what is the numbers `50` and `20`? add constants for them

**Review by OlEzhA0 — APPROVED** [review]
Hey! Good Job! I approve your work but anyway pay attention to comments

**Pull request closed without merging**